tests/foreman/api/test_provisioningtemplate.py

```diff
@@ -452,9 +452,11 @@ def test_positive_template_use_graphical_installer(
         assert 'skipx' not in render
         assert 'chvt 6' in render
 
-    @pytest.mark.rhel_ver_match('[8]')
+    @pytest.mark.rhel_ver_list([settings.content_host.default_rhel_version])
```

Comment:
@Gauravtalreja1 Should we test the upgrade too?

Reply:
It is just a provisioning template test, where we create a new host, and I don't see any value addition

Reply:
Please address rest of the changes to test 2.5 in another PR as discussed.

Reply:
I am good with skipping this test for upgrade.